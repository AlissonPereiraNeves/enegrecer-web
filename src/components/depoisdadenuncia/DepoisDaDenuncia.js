import React from 'react';
import imagemMaos from '../../imagens/depois-denuncia-1.jpg';
import imagemHomemNegro from '../../imagens/depois-denuncia-2.jpg';
import imagemMulherNegra from '../../imagens/depois-denuncia-3.jpg';
import SecaoDaPagina from './componentes/SecaoDaPagina';

const DepoisDaDenuncia = () => (
  <div>
    <SecaoDaPagina
      tituloDaSecao1="E depois"
      tracoVermelho="bloco_retangular"
      tituloDaSecao2="da denúncia?"
      textoDoBloco="Lorem ipsum dolor sit amet, consectetur adipiscing elit.
            Quisque eget lectus eget ex viverra laoreet. Sed vitae massa
            vitae tortor pretium rutrum. Fusce in mattis urna, a lobortis
            elit. Suspendisse id maximus neque, non tincidunt ex. Mauris
            justo neque, finibus eget posuere at, pharetra a lectus. Etiam
            at risus sit amet nunc commodo ullamcorper vel non enim. Ut
            placerat scelerisque fermentum. Etiam fermentum convallis ipsum
            eget dapibus. Cras ut libero et dolor eleifend bibendum. Phasellus
            maximus nibh in sem congue cursus. Fusce vitae mi quis magna
            rutrum ultricies. Aenean ullamcorper dapibus lacinia. Lorem ipsum
            dolor sit amet, consectetur adipiscing elit. Quisque eget lectus
            eget ex viverra laoreet. Sed vitae massa vitae tortor pretium
            rutrum. Fusce in mattis urna, a lobortis elit."
      ladoTexto="esquerdo"
      imagemSecao={imagemMaos}
      nomeFoto="maos de pessoas negras"
    />

    <SecaoDaPagina
      nomeDaClasse="segundo__bloco-proximos-passos"
      textoDoBloco="Lorem ipsum dolor sit amet, consectetur adipiscing elit.
            Quisque eget lectus eget ex viverra laoreet. Sed vitae massa
            vitae tortor pretium rutrum. Fusce in mattis urna, a lobortis
            elit. Suspendisse id maximus neque, non tincidunt ex. Mauris
            justo neque, finibus eget posuere at, pharetra a lectus. Etiam
            at risus sit amet nunc commodo ullamcorper vel non enim. Ut
            placerat scelerisque fermentum. Etiam fermentum convallis ipsum
            eget dapibus. Cras ut libero et dolor eleifend bibendum. Phasellus
            maximus nibh in sem congue cursus. Fusce vitae mi quis magna
            rutrum ultricies. Aenean ullamcorper dapibus lacinia. Lorem ipsum
            dolor sit amet, consectetur adipiscing elit. Quisque eget lectus
            eget ex viverra laoreet. Sed vitae massa vitae tortor pretium
            rutrum. Fusce in mattis urna, a lobortis elit."
      ladoTexto="direito"
      imagemSecao={imagemHomemNegro}
      nomeFoto="homem negro"
    />

    <SecaoDaPagina
      nomeDaClasse="terceiro__bloco-proximos-passos"
      tituloDaSecao1="E depois"
      tracoVermelho="bloco_retangular"
      tituloDaSecao2="da denúncia?"
      textoDoBloco="Lorem ipsum dolor sit amet, consectetur adipiscing elit.
            Quisque eget lectus eget ex viverra laoreet. Sed vitae massa
            vitae tortor pretium rutrum. Fusce in mattis urna, a lobortis
            elit. Suspendisse id maximus neque, non tincidunt ex. Mauris
            justo neque, finibus eget posuere at, pharetra a lectus. Etiam
            at risus sit amet nunc commodo ullamcorper vel non enim. Ut
            placerat scelerisque fermentum. Etiam fermentum convallis ipsum
            eget dapibus. Cras ut libero et dolor eleifend bibendum. Phasellus
            maximus nibh in sem congue cursus. Fusce vitae mi quis magna
            rutrum ultricies. Aenean ullamcorper dapibus lacinia. Lorem ipsum
            dolor sit amet, consectetur adipiscing elit. Quisque eget lectus
            eget ex viverra laoreet. Sed vitae massa vitae tortor pretium
            rutrum. Fusce in mattis urna, a lobortis elit."
      ladoTexto="esquerdo"
      imagemSecao={imagemMulherNegra}
      nomeFoto="mulher negra"
    />
  </div>
);

export default DepoisDaDenuncia;
