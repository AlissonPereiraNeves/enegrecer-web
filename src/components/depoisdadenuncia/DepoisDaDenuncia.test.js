import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import DepoisDaDenuncia from './DepoisDaDenuncia';

describe('Depois da denuncia', () => {
  const wrapper = shallow(<DepoisDaDenuncia />);

  it('possui a mesma estrutura', () => {
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
  });
});
