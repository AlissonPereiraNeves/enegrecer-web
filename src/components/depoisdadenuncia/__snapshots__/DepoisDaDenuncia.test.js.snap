// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Depois da denuncia possui a mesma estrutura 1`] = `
<div>
  <SecaoDaPagina
    imagemSecao="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fdepois-denuncia-1.jpg?alt=media&token=66d1b719-80cf-4b08-97e8-88d1b9ccdce9"
    ladoTexto="esquerdo"
    nomeFoto="maos de pessoas negras"
    textoDoBloco="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque eget lectus eget ex viverra laoreet. Sed vitae massa vitae tortor pretium rutrum. Fusce in mattis urna, a lobortis elit. Suspendisse id maximus neque, non tincidunt ex. Mauris justo neque, finibus eget posuere at, pharetra a lectus. Etiam at risus sit amet nunc commodo ullamcorper vel non enim. Ut placerat scelerisque fermentum. Etiam fermentum convallis ipsum eget dapibus. Cras ut libero et dolor eleifend bibendum. Phasellus maximus nibh in sem congue cursus. Fusce vitae mi quis magna rutrum ultricies. Aenean ullamcorper dapibus lacinia. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque eget lectus eget ex viverra laoreet. Sed vitae massa vitae tortor pretium rutrum. Fusce in mattis urna, a lobortis elit."
    tituloDaSecao1="E depois"
    tituloDaSecao2="da denúncia?"
    tracoVermelho="bloco_retangular"
  />
  <SecaoDaPagina
    imagemSecao="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fdepois-denuncia-2.jpg?alt=media&token=7b53de21-63a8-440a-a1a2-ae57121c8034"
    ladoTexto="direito"
    nomeDaClasse="segundo__bloco-proximos-passos"
    nomeFoto="homem negro"
    textoDoBloco="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque eget lectus eget ex viverra laoreet. Sed vitae massa vitae tortor pretium rutrum. Fusce in mattis urna, a lobortis elit. Suspendisse id maximus neque, non tincidunt ex. Mauris justo neque, finibus eget posuere at, pharetra a lectus. Etiam at risus sit amet nunc commodo ullamcorper vel non enim. Ut placerat scelerisque fermentum. Etiam fermentum convallis ipsum eget dapibus. Cras ut libero et dolor eleifend bibendum. Phasellus maximus nibh in sem congue cursus. Fusce vitae mi quis magna rutrum ultricies. Aenean ullamcorper dapibus lacinia. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque eget lectus eget ex viverra laoreet. Sed vitae massa vitae tortor pretium rutrum. Fusce in mattis urna, a lobortis elit."
    tituloDaSecao1=""
    tituloDaSecao2=""
    tracoVermelho=""
  />
  <SecaoDaPagina
    imagemSecao="https://firebasestorage.googleapis.com/v0/b/enegrecer-e37b3.appspot.com/o/fotos%2Fdepois-denuncia-3.jpg?alt=media&token=e5d639c9-6dd9-45f6-94b0-847edd65e432"
    ladoTexto="esquerdo"
    nomeDaClasse="terceiro__bloco-proximos-passos"
    nomeFoto="mulher negra"
    textoDoBloco="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque eget lectus eget ex viverra laoreet. Sed vitae massa vitae tortor pretium rutrum. Fusce in mattis urna, a lobortis elit. Suspendisse id maximus neque, non tincidunt ex. Mauris justo neque, finibus eget posuere at, pharetra a lectus. Etiam at risus sit amet nunc commodo ullamcorper vel non enim. Ut placerat scelerisque fermentum. Etiam fermentum convallis ipsum eget dapibus. Cras ut libero et dolor eleifend bibendum. Phasellus maximus nibh in sem congue cursus. Fusce vitae mi quis magna rutrum ultricies. Aenean ullamcorper dapibus lacinia. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque eget lectus eget ex viverra laoreet. Sed vitae massa vitae tortor pretium rutrum. Fusce in mattis urna, a lobortis elit."
    tituloDaSecao1="E depois"
    tituloDaSecao2="da denúncia?"
    tracoVermelho="bloco_retangular"
  />
</div>
`;
